src/qililab/execution/execution_builder.py

```diff
@@ -98,5 +98,5 @@ def _get_bus_info_from_loop_alias(self, platform: Platform, loop: Loop):
             bus: Bus object
         """
         alias = loop.alias
-        bus = platform._get_bus_by_alias(alias=alias)  # pylint: disable=protected-access
+        bus = platform.get_bus_by_alias(alias=alias)  # pylint: disable=protected-access
```

Comment:
If I remember correctly, @fedonman made this private in a recent PR.

Even though this is used outside of the class, I like hiding it from the user, given that they should not use it.